config/kinto.ini

```diff
@@ -1,35 +1,37 @@
+[server:main]
+use = egg:waitress#main
+host = 0.0.0.0
+port = 8888
+
 [app:main]
 use = egg:kinto
-
-pyramid.debug_notfound = true
-
 kinto.http_scheme = http
-kinto.http_host = localhost:8888
-
-# kinto.project_docs = https://kinto.readthedocs.org/
+kinto.http_host = 0.0.0.0:8888
 
 #
 # Backends.
 #
-# kinto.cache_backend = cliquet.cache.postgresql
-# kinto.cache_url = postgres://postgres:postgres@localhost/postgres
+# http://kinto.readthedocs.org/en/latest/configuration/settings.html#storage
+#
 # kinto.storage_backend = cliquet.storage.postgresql
 # kinto.storage_url = postgres://postgres:postgres@localhost/postgres
+# kinto.cache_backend = cliquet.cache.postgresql
+# kinto.cache_url = postgres://postgres:postgres@localhost/postgres
 # kinto.permission_backend = cliquet.permission.postgresql
 # kinto.permission_url = postgres://postgres:postgres@localhost/postgres
 
-# kinto.backoff = 10
-# kinto.batch_max_requests = 25
-# kinto.retry_after_seconds = 30
-# kinto.eos =
-
 #
 # Auth configuration.
 #
+# http://kinto.readthedocs.org/en/latest/configuration/settings.html#authentication
+#
 kinto.userid_hmac_secret = b4c96a8692291d88fe5a97dd91846eb4
```

Comment:
The [documentaton](http://kinto.readthedocs.org/en/latest/configuration/settings.html#authentication) explicitly states
> By default, Kinto doesn’t define a secret for you

Perhaps this value should be empty?